web/components/TextBlock/TextBlock/TextBlock.tsx

```diff
@@ -42,7 +42,7 @@ const components: Partial<PortableTextComponents> = {
     underline: ({ children }) => <u>{children}</u>,
     code: ({ children }) => <code>{children}</code>,
     internalLink: ({ text, value }) => (
-      <Link href={getEntityPath(value.reference)}>{text}</Link>
+      <Link href={value?.href || getEntityPath(value)}>{text}</Link>
```

Comment:
Nit: The href is OK, but replacing `value.reference` with `value` makes no difference that I can see. Neither of those values is actually an `Entity`. Loading the entire entity for every internal link instead of just its slug also sounds pretty heavy anyway.

Besides, the entire `getEntityPath` helper seems to be outdated ever since https://github.com/sanity-io/structured-content-2022/pull/49 was merged. It handles `session`, `person` and `venue` by creating URLs that are no longer handled. But the possible references in the Studio schema are actually `route`, `person`, `session` and `article`, where at least `route` and `article` might be expected to work at this point.

But this might be considered out of scope for this PR.

Reply:
Yeah, the property deletion was accidental, reverted in adaf179. I don't really know what to do with `getEntityPath` at this point, so I'll just leave it as-is. I filed #96 for it.